src/main/java/com/lpvs/auth/OAuthService.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2023 Basaeng, kyudori, hwan5180, quswjdgma83
+ *
+ * Use of this source code is governed by a MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package com.lpvs.auth;
+
+import com.lpvs.entity.LPVSMember;
+import com.lpvs.repository.LPVSMemberRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.oauth2.client.userinfo.DefaultOAuth2UserService;
+import org.springframework.security.oauth2.client.userinfo.OAuth2UserRequest;
+import org.springframework.security.oauth2.client.userinfo.OAuth2UserService;
+import org.springframework.security.oauth2.core.OAuth2AuthenticationException;
+import org.springframework.security.oauth2.core.user.DefaultOAuth2User;
+import org.springframework.security.oauth2.core.user.OAuth2User;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+@Service
+@RequiredArgsConstructor
+public class OAuthService implements OAuth2UserService<OAuth2UserRequest, OAuth2User> {
+
+    private final LPVSMemberRepository LPVSMemberRepository;
```

Comment:
It is better to name var starting with a lowercase letter, and it should be different from class name. I'd suggest to name `lpvsMemberRepository `or just `memberRepo`

Reply:
Done.